//package com.sincetimes.redwar.game.comnunicate.protocol;
message RseQueryVisitHelpsGiftUnitsOnBunker
{
	optional int32 ret = 1;
}